src/connectors/youtube-music.js

```diff
@@ -26,6 +23,22 @@ Connector.albumSelector = [
 	'.ytmusic-player-bar .yt-formatted-string.style-scope.yt-simple-endpoint[href*="browse/FEmusic_library_privately_owned_release_detailb_"]',
 ];
 
+Connector.getArtistTrack = () => {
+	let artist; let track;
+	if (Connector.getAlbum()) {
+		Connector.resetFilter();
```

Comment:
Calling `Connector.resetFilter` will create a new filter instance on every call.

Reply:
Since the metadata filter package mutates the filter object in `extend`, I'm not sure if there's a good way to avoid this without saving a copy of the default filter and then deep cloning on every reset. Thoughts?

https://github.com/web-scrobbler/metadata-filter/blob/67f7fb53b97edc34c07946e5fb0d8c133adac7ce/src/filter.js#L63-L66